Automattic/crowdsignal-ui
# Include new Patterns list

## Summary

This PR adds the necessary changes to (finally 🎉 ) make the new pattern list work.

## Testing Instructions
* Open or create a project
* Click the [ + ] icon to open the Block Library
* Select the Patterns tab
* Check if the new pattern list is available
* Play with the patterns to make sure they are working properly

**Note:** There are some styling glitches with the pattern preview that make the preview taller than necessary. That is not related to these changes, and we'll try to address it in a separate PR.
